CMEW/flow.cylc

```diff
@@ -85,14 +100,14 @@
             ROOTPATH_RAWOBS = {{ ROOTPATH_RAWOBS }}
 
     [[configure_for<recipe>]]
-        inherit = None, RECIPE
+        inherit = None, RECIPE, TWO_RUNS
         [[[environment]]]
             ROSE_TASK_APP = configure_for
             START_YEAR = {{ START_YEAR }}
             NUMBER_OF_YEARS = {{ NUMBER_OF_YEARS }}
 
     [[configure_standardise<recipe>]]
-        inherit = STANDARDISE, RECIPE
+        inherit = STANDARDISE, RECIPE, TWO_RUNS
         [[[environment]]]
             ROSE_TASK_APP = configure_standardise
             CALENDAR = {{ CALENDAR }}
```

Comment:
```suggestion
```
We don't need to retain separate variables here for `CALENDAR`, `MODEL_ID` and `SUITE_ID`. Removing them from here will enable my naming suggestion above (lines 70-73).